Libraries/Components/TextInput/TextInput.js

```diff
@@ -301,6 +301,13 @@ type IOSProps = $ReadOnly<{|
    * @platform ios
    */
   textContentType?: ?TextContentType,
+
+  /**
+   * If `true`, it will hide the done button above the keyboard
+   * Only work for any number pads keyboardType
+   * @platform ios
+   */
+   hideDoneButtonIOS?: ?boolean,
```

Comment:
1. Please rename it to `showDoneButtonForNumberPadKeyboard`.
2. Remove the negation it the place where this value is used.
3. Update the above description.
4. Delete extra whitespace before this property name.

Reply:
@dmitryrykun thank you for reviewing this PR. I have done the changes as mentioned but couldn’t figure it out to set the default value of showDoneButtonForNumberPadKeyboard to true.

Reply:
@vikassharma96  that's great, `initWithBridge` is usually the best place to initialise object properties. Please push your changes.